Wrap castor in a docker container (#65)
jolicode/castor: README.md

ln -s $PWD/bin/castor $HOME/.local/bin/castor
```

### With docker

If PHP is installed on your system, you can use the `castor` PHAR directly.

> **This is the recommended way** because some features does't work correctly,

Reviewer: ```suggestion
**This is the recommended way** because some features require to be executed on the host to work correctly,
```

like notification. However, if you don't have PHP installed, you can use docker.

We ship a `Dockerfile` that you can use to build a docker image with castor:

```
docker build -t castor .
```

Then you can run castor with:

```
docker run -it --rm -v `pwd`:/project castor
```

> If you want to use docker commands in your tasks, you must enabled docker

Reviewer: ```suggestion
If you want to use docker commands in your tasks, you must enable docker
```

support when building the image:

```
docker build -t castor --build-arg WITH_DOCKER=1  .
```

Then you can run castor with:

```
docker run -it --rm -v `pwd`:/project -v "/var/run/docker.sock:/var/run/docker.sock:rw" castor
```

> We suggest you to created an alias for it:

Reviewer: ```suggestion
We suggest you to create an alias for it:
```
